sources/packages/backend/workflow/test/2022-2023/parttime-assessment/appeals/parttime-assessment-student-financial-information.e2e-spec.ts

```diff
@@ -97,6 +97,73 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-student-financia
     },
   );
 
+  it(
+    "Should have calculated student total income from the current year income and not from the the application data or the tax return income " +
+      "when there is a request a change for student financial information with current year income values.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataHasDependents = YesNoOptions.No;
+      assessmentConsolidatedData.appealsStudentFinancialInformationAppealData =
+        {
+          taxReturnIncome: 1234,
+          currentYearIncome: 500,
+        };
+
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+
+      // Assert
+      expect(
+        calculatedAssessment.variables.calculatedDataCurrentYearIncome,
+      ).toBe(500);
+      expect(
+        calculatedAssessment.variables.calculatedDataTotalFamilyIncome,
+      ).toBe(500);
+      expect(
+        calculatedAssessment.variables.calculatedDataStudentTotalIncome,
+      ).toBe(500);
+    },
+  );
+
+  it(
+    "Should have calculated student total income from the tax return income and not from the the application data tax return income " +
+      "when there is a request a change for student financial information with tax return income values.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataCRAReportedIncome = null;
+      assessmentConsolidatedData.studentDataHasDependents = YesNoOptions.No;
+      assessmentConsolidatedData.appealsStudentFinancialInformationAppealData =
+        {
+          taxReturnIncome: 1234,
+        };
+
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+
+      // Assert
+      //Application data tax return income submitted by the student.
+      expect(assessmentConsolidatedData.studentDataTaxReturnIncome).toBe(40001);
```

Comment:
Is this assertion part of the test scope?

Reply:
just to show case, that the student has inout TaxReturn income in the application data and its not considered.